rpcserver.go

```diff
@@ -377,29 +402,24 @@ func (r *rpcServer) MintAsset(ctx context.Context,
 		AssetType:      asset.Type(req.Asset.AssetType),
 		AssetName:      req.Asset.Name,
 		Amount:         req.Asset.Amount,
-		EnableEmission: req.EnableEmission,
+		EnableEmission: req.NewGroupedAsset,
```

Comment:
Could also change the attribute to match as well.

Reply:
Attribute as in field in `tapgarden.Seedling` ?